src/mpid/ch3/channels/nemesis/include/mpid_nem_datatypes.h

```diff
@@ -116,12 +116,12 @@
    
 */
 
-#if (SIZEOF_OPA_PTR_T > 8)
```

Comment:
How is this not causing a compile error?

Reply:
Good question. I only know it was set to 0.